app/models/role.rb

```diff
@@ -30,13 +30,25 @@ def organisations
     links["ordered_parent_organisations"]
   end
 
+  def current_holder
+    links["role_appointments"]
```

Comment:
```suggestion
    links.fetch("role_appointments", [])
```

Although unlikely, it's possible that a ministerial role could be created and not yet assigned any people.

Reply:
I thought that wasn't possible. 🤔 I'll change it.